app/controllers/settings_controller.rb

```diff
@@ -4,5 +4,7 @@ class SettingsController < ApplicationController
 
   def edit
     @user = current_user
+    @webauthn_credentials = @user.webauthn_credentials
+    @webauthn_credential = WebauthnCredential.new(user: @user)
```

Comment:
`@user.webauthn_credentials.build` adds an extra webauthn credential to the user object, so I think it's better to do it this way